rochambeau.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Rochambeau</title>
+    <link rel="stylesheet" type="text/css" href="src/main.css">
+    <link rel="stylesheet" type="text/css" href="src/rochambeau.css">
+</head>
+
+<body>
+
+    <header>
+        <nav>
+            <a href="./">Home</a>
+            <a href="guess.html">Play the Numbers!</a>
+            <a href="rochambeau.html">Rochambeau</a>
+        </nav>
+
+        <h1>ROCHAMBEAU</h1>
+    </header>
+
+    <main>
+        <h2>The Classic Game of Rock, Paper, Scissors</h2>
+
+        <ul>
+            <li>Rock beats Scissors</li>
+            <li>Scissors beats Paper</li>
+            <li>Paper beats Rock</li>
+        </ul>
+
+        <h2>Click an image to play:</h2>
+
+        <section>
+            <!-- //changed radio buttons to regular buttons to combine functions -->
```

Comment:
If you're changing from radio buttons to normal buttons, you'll want to refactor at the html for these! Three separate normal buttons won't need the type property and shared name. Take a look at the docs for buttons and you'll be able to see what you do/don't need. Otherwise, I like that you approached this problem in a different way!